trust/src/all.rs

```diff
@@ -0,0 +1,77 @@
+use std::process::ExitCode;
+
+#[derive(clap::Args, Debug)]
+pub struct Command {
+    #[arg(long = "bombastic-api-disabled", default_value_t = false)]
+    pub(crate) bombastic_api_disabled: bool,
```

Comment:
Personally, I like the `--no-...` convention, e.g. `--no-bombastic-api`, but that prefix usually indicates that `--bombastic-api` would also be a valid option, which implies sort of a different cli: `all --no-vex-indexer` or `none --bombastic-api --bombastic-indexer`. Maybe overkill? Dunno. :smile: 

BTW, I checked the clap repo and there are a few issues around supporting the `--no-` prefix, but nothing yet implemented, so far as I can tell.